apps/mobile/src/features/send/send-sheet.tsx

```diff
@@ -28,23 +35,25 @@ export function SendSheet() {
     analytics.track('send_sheet_dismissed');
   }
 
+  useImperativeHandle(sendSheetRef, () => ({
+    present(newAsset) {
+      setAsset(newAsset);
+      ref.current?.present();
+    },
+    dismiss() {
+      ref.current?.dismiss();
+    },
+  }));
+
   return (
     <FullHeightSheet
-      sheetRef={sendSheetRef}
+      sheetRef={ref}
       onAnimate={handleAnimatedPositionChange}
       onDismiss={handleDismiss}
     >
-      <Send currentAccount={currentAccount} asset={asset} />
+      <SheetNavigationContainer base="send">
+        <Send currentAccount={currentAccount} asset={asset} />
+      </SheetNavigationContainer>
     </FullHeightSheet>
   );
 }
-
-function useInitialSendParams() {
```

Comment:
We already have access to the `asset` when the user is clicking `Send` from token details so it's easier to decouple this sheet from the routing and use `useImperativeHandle` similar to the receive sheet.